tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = style, unit
+
+# Configs
+[pytest]
+addopts = -p no:warnings
+
+# Local Unit
+[testenv:unit]
+deps =
+    codecov
+    mock
+    pytest
+    pytest-cov
+    pytest-mock
+commands =
+    pytest tests
+
+# Codestyle
+[testenv:style]
+deps = flake8
+commands = flake8 --ignore E501, crossengage
```

Comment:
Instead of ignoring E501, I think it is better to set a longer line length, like 120 for example.